### ISSUE TITLE
Don't explicitly load default

### ISSUE THREAD

**Issue by Wilfred** [mohebifar/lebab#164]
If I'm writing ES5 trying to load code defined in ES6, I end up doing this:

```js
var jsdomify = require('jsdomify').default;
```

which lebab rewrites as:

```js
import {default as jsdomify} from 'jsdomify';
```

however, it could be simply:

```js
import jsdomify from 'jsdomify';
```

**Comment by apexearth** [comment]
This looks like it is importing from a structure like the following.

```
{ 
  default: something...
}
```

So your default export has a property on it called default.

**Comment by nene** [comment]
Actually [jsdomify](https://github.com/dmatteo/jsdomify) is doing a proper default export:

```js
export default {
  create,
  clear,
  destroy,
  getDocument
};
```

Babel will compile this something like:

```
exports.default = {create, clear, destroy, getDocument};
```

That's a new default behavior since Babel6. One can change this by using [add-module-exports](https://www.npmjs.com/package/babel-plugin-add-module-exports) plugin. Lebab itself uses this to expose a simpler programming API.

From ES6 perspective however, this:

```js
import {default as jsdomify} from 'jsdomify';
```

is exact equivalent to:

```js
import jsdomify from 'jsdomify';
```

So yeah... Lebab should transform into the latter.

**Comment by nene** [comment]
Fixed and released in 2.1.1
